### DISCUSSION TITLE
Python Files

### DISCUSSION THREAD

**Discussion by kokonut27** [ReplAPI-it/ReplAPI.it-Python#10] [Package Help] [answered]
Hiya, quick question;

Where did the python files go? ;-;

Thanks! :D

**Comment by rayhanadev** [comment]
No idea 🤷‍♂️.

Also probs use Team Discussion or Repl Threads over Repo Issue for questions.

**Comment by kokonut27** [comment, marked as answer]
Okie :)

Oh, just saw it xD I'll use that later on :D

**Reply by kokonut27** [reply]
Closing and marking answer as this has been fixed.
